Makefile

```diff
@@ -21,7 +21,7 @@ TAG ?= latest
 IMAGE_REF=$(REGISTRY)/$(TARGET_NAME):$(TAG)
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
 IMAGE_REGISTRY?=registry.svc.ci.openshift.org
-KUBEVIRT_PROVIDER?=k8s-1.26
+KUBEVIRT_PROVIDER?=k8s-1.28
```

Comment:
I see that this KUBEVIRT_PROVIDER env var is all over the place now, but is converged to 1.28 everywhere.

Do you think it is practical for us to have a single place where this env var is set? maybe `kubevirtci`. That would help when we eventually need to increment this variable again in the future

Reply:
The problem is that the prow jobs call the hack/xxx.sh scripts directly and they cannot inherit the env variable from the Makefile, where it is defaulted. I suppose if we make the proper make targets for each type of test, then we can define it there.

Reply:
what if the env var is only in the kubevirtci script, would that work?

Reply:
That seems to work, I modified the PR to do that.